pandas/tests/test_base.py

```diff
@@ -730,11 +726,11 @@ def test_duplicated_drop_duplicates(self):
                 tm.assert_index_equal(result, idx[~expected])
 
                 # deprecate take_last
-                with tm.assert_produces_warning(FutureWarning):
+                with tm.assert_produces_warning(FutureWarning, clear=pd.core.base):
```

Comment:
we don't normally pass ``clear``. seems this should in fact default to the current module (but can leave that for another PR).